### ISSUE TITLE
Consistency on MASS names

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#48]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/48) by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Check this out:

```
#!bash

fowlerj@817jfmac:mass$ grep "chan_num" `find mass -name "*.py"`|wc
      26     160    2270
fowlerj@817jfmac:mass$ grep "channum" `find mass -name "*.py"`|wc
     191    1402   22040

```

So this means that *channum* appears 191 times in MASS and *chan_num* 26 times. Some of these are internal variable names and don't matter, but some are certainly named parameter arguments. We need consistency on whether to spell this with underscores. Ugh! On the other hand, it's a breaking change. We'll have to figure out a way to do it, perhaps as part of a larger reorganization.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

This is too broad of an issue. Let's just change this specific item as part of Mass 0.5.0.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Rename chan_num -> channum in TESGroup.read_trace() and .plot_traces()

Fixes #48.
